docs/data/external-oasis-repos.md

```diff
@@ -156,7 +156,7 @@ In order to host a repository on OASIS, perform the following steps:
     will be contacted in case of issues. To designate an OASIS manager, have the VO manager update the
     [Topology registration](https://github.com/opensciencegrid/topology/tree/master/virtual-organizations).
 
-1.  Create a [support ticket](https://support.opensciencegrid.org/helpdesk/tickets/new) using the following template:
+1.  Send a message to [OSG support](mailto:support@osg-htc.org) using the following template:
```

Comment:
```suggestion
1.  Send a message to [OSG support](mailto:help@osg-htc.org) using the following template:
```

Reply:
help@osg-htc.org is what we list on our help page (https://osg-htc.org/docs/common/help/)